### PR TITLE
add ptp operator

### PR DESCRIPTION
**Pull request by jnunyez**

### CODE REVIEW

File: components/apps/ptp-operator/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# OpenShift Precision Time Protocol Operator
+
+Installs the OpenShift PTP Operator.
+
+Do not use the `base` directory directly, as you will need to patch the `channel` based on the version of OpenShift you are using, or the version of the operator you want to use.
```
Comment (novacain1) [inline components/apps/ptp-operator/README.md]: I thought with 4.10 we moved to a "stable" channel distinction, versus a version number that has to increment every minor release?

File: components/apps/ptp-operator/base/subscription.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: ptp-operator-subscription
+  namespace: openshift-ptp
+spec:
+  channel: patch-me
+  name: ptp-operator
+  source: redhat-operators
```
Comment (novacain1) [inline components/apps/ptp-operator/base/subscription.yaml:10]: This looks good for now, but consider disconnected use cases where we mirror the catalog and need to adjust the catalogsource.

**Review by novacain1 — COMMENTED** [review]
Just a few comments here, let me know what you think.

File: components/apps/ptp-operator/README.md (hunk 1 above)
Comment (jnunyez) [reply components/apps/ptp-operator/README.md]: Agree in that "stable" is going to be default channel no matter the OCP release. However in the near future we might be interested on getting some preview, fast channel, if needed. Gonna correct the README.md.

File: components/apps/ptp-operator/base/subscription.yaml (hunk 2 above)
Comment (jnunyez) [reply components/apps/ptp-operator/base/subscription.yaml:10]: Noted.

File: components/apps/ptp-operator/README.md (hunk 1 above)
Comment (novacain1) [reply components/apps/ptp-operator/README.md]: +1

**Pull request merged**